Packs/GuardiCore/Integrations/GuardiCoreV2/GuardiCoreV2.py

```diff
@@ -333,18 +333,21 @@ def fetch_incidents(client: Client, args: Dict[str, Any]) -> \
             demisto.debug(
                 f'{INTEGRATION_NAME} - Fetch incidents: skipped fetched incident because no start time or id')
             continue
-
-        incident = {
-            'name': f"{INTEGRATION_CONTEXT_NAME} Incident (INC-{id.split('-')[0].upper()})",
-            'occurred': timestamp_to_datestring(start_time, DATE_FORMAT),
-            'severity': incident_severity_to_dbot_score(severity),
-            'rawJSON': json.dumps(inc)
-        }
-        incidents.append(incident)
-
+        demisto.debug(
+            f"Fetch incident checking id: {id} with start time of:"
+            f" {start_time}. LastRun time was: {current_fetch}")
         if current_fetch < start_time:
```

Comment:
is it guaranteed that the results are retrieved in ascending order?
e.g. if current_fetch = 10 and we got two incidents, the first has current_time=12 and the second has current_time=11 - you won't fetch the second one.
please check this, and if needed, either always use the first value of current_fetch, or sort the results.

Reply:
The customer approved the fix but we can add `{sort: descending}` to the fetch params. (but we don't have an instance to check it out)